src/Sylius/Bundle/OrderBundle/Doctrine/ORM/OrderRepository.php

```diff
@@ -115,16 +116,25 @@ public function findCartById($id): ?OrderInterface
         ;
     }
 
-    public function findCartsNotModifiedSince(\DateTimeInterface $terminalDate): array
+    public function findCartsNotModifiedSince(\DateTimeInterface $terminalDate, ?int $limit = null): array
     {
-        return $this->createQueryBuilder('o')
+        if (null === $limit) {
+            trigger_deprecation('sylius/order-bundle', '1.13', 'Not passing a limit to "%s()" is deprecated and will be removed in Sylius 2.0.', __METHOD__);
```

Comment:
You are adding a deprecation to 1.11 from 1.13 🤔

Reply:
I removed this trigger.